### DISCUSSION TITLE
Spreadsheet Style fast editing

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [keystonejs/keystone#5534] [Feature requests]
Keystone should provide a List Editor option for Spreadsheet style data updation. 

By Spreadsheet style editing, I mean the rapid view, comparison, updation of values
a. Look at tax rates for a number of products
b. Update the tax rates for multiple productions at one shot through a ````drag down```` operation

Use Cases where Spreadsheet style editing will be useful
* ecommerce sites
* shipping tables

**Comment by bladey** [comment]
Thanks for the idea @VinayaSathyanarayana!
